rasgotransforms/rasgotransforms/transforms/filter/filter.sql

```diff
@@ -16,6 +16,8 @@ FROM {{ source_table }}
 {%- else -%}
     {%- if filter_block['operator'] == 'CONTAINS' -%}
 {{ filter_block['operator'] }}({{ filter_block['columnName'] }}, {{ filter_block['comparisonValue'] }})
+    {%- elif filter_block['operator'] == 'IN' -%}
+        {{ filter_block['columnName'] }} {{ filter_block['operator'] }} ({{ filter_block['comparisonValue'] }})
```

Comment:
```suggestion
    {%- elif filter_block['operator'] == 'IN' -%}
        {{ filter_block['columnName'] }} IN ({{ filter_block['comparisonValue'] }})
```
For sections like these curious if it might be more readable to just put the operator in them directly. (Same is true for the CONTAINS as well.)